README.md

```diff
@@ -1,2 +1,60 @@
-# Leaderboard
-Just a leaderboard web app
+# Leaderboard web app
+An API-driven web app to display a simple scoreboard.
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+
+![screenshot](./screenshots/screenshot.png)
```

Comment:
- [ ] This image is broken, please fix it to display a more professional readme.